libraries/AP_HAL_ChibiOS/hwdef/BlitzF745/defaults.parm

```diff
@@ -0,0 +1,3 @@
+# setup for LEDs on chan9
+SERVO9_FUNCTION 120
+NTF_LED_TYPES 257
```

Comment:
since you have DisplayPort setup, might as well enable its OSD use which can coexist with analog OSD
```suggestion
NTF_LED_TYPES 257
OSD_TYPE2 5
```

Reply:
Thanks fixed